docs/configuration.md

```diff
@@ -0,0 +1,394 @@
+| :------ | :--------------------------------------------------------------------------------------------------- | :----- |
+| address | libp2p formatted address(PeerID can be found on the dashboard or in the logs. It starts with `12D3`) | string |
+| alias   | alias of the peer                                                                                    | string |
+
+Example:
+
+```toml
+[network]
+bind_address             = "/ip4/0.0.0.0/tcp/15600"
+reconnect_interval_secs  = 30
+max_unknown_peers        = 4
+
+[network.peering]
+[[network.peering.peers]]
+address = "/ip4/192.0.2.0/tcp/15600/p2p/PeerID"
+alias   = "some peer"
+[[network.peering.peers]]
+address = "/ip6/2001:db8::/tcp/15600/p2p/PeerID"
+alias   = "another peer"
+[[network.peering.peers]]
+address = "/dns/example.com/tcp/15600/p2p/PeerID"
+alias   = "yet another peer"
+```
+
+## Protocol
+
+| Name                        | Description           | Type         |
+| :-------------------------- | :-------------------- | :----------- |
+| minimum_pow_score           | the minimum pow score | float[f64]   |
+| [coordinator](#coordinator) | coordinator configs   | table        |
+| [workers](#workers)         | worker configs        | table        |
+
+### coordinator
+
+| Name                                    | Description           | Type            |
+| :-------------------------------------- | :-------------------- | :-------------- |
+| public_key_count                        | number of public keys | integer[usize]  |
+| [public_key_ranges](#public_key_ranges) | public key ranges     | array of tables |
+
+#### public_key_ranges
+
+| Name       | Description | Type         |
+| :--------- | :---------- | :----------- |
+| public_key | public key  | string       |
+| start      | start       | integer[u32] |
+| end        | end         | integer[u32] |
+
+### workers
+
+| Name                 | Description           | Type           |
+| :------------------- | :-------------------- | :------------- |
+| message_worker_cache | TO-DO                 | integer[usize] |
+| status_interval      | status interval in ms | integer[u64]   |
+| ms_sync_count        | milestone sync count  | integer[u32]   |
+
+Example:
+
+```toml
+[protocol]
+minimum_pow_score = 4000
+handshake_window = 10
+[protocol.coordinator]
+public_key_count  = 2
+[[protocol.coordinator.public_key_ranges]]
+public_key  = "7205c145525cee64f1c9363696811d239919d830ad964b4e29359e6475848f5a"
+start       = 0
+end         = 0
+[[protocol.coordinator.public_key_ranges]]
+public_key  = "e468e82df33d10dea3bd0eadcd7867946a674d207c39f5af4cc44365d268a7e6"
+start       = 0
+end         = 0
+[[protocol.coordinator.public_key_ranges]]
+public_key  = "0758028d34508079ba1f223907ac3bb5ce8f6bdccc6b961c7c85a2f460b30c1d"
+start       = 0
+end         = 0
+[protocol.workers]
+message_worker_cache = 10000
+status_interval = 10
+ms_sync_count = 200
+```
+
+## Rest_api
```

Comment:
REST API

Reply:
How did I revert that one? Will fix it again ;)